pyproject.toml

```diff
@@ -34,13 +34,13 @@ methoddispatch = "^3.0.2"
 msgpack = "^1.0.0"
 httpx = [
   { version = "^0.24.1", python = "~3.7" },
-  { version = "^0.25.0", python = "^3.8" },
+  { version = ">= 0.25.0, < 1.0", python = "^3.8" },
 ]
 h2 = "^4.1.0" # required for httx package, HTTP2 communication
 websockets = ">= 10.0, < 13.0"
 pyee = [
   { version = "^9.0.4", python = "~3.7" },
-  { version = "^11.1.0", python = "^3.8" }
+  { version = ">= 11.1.0, < 12.0", python = "^3.8" }
```

Comment:
what's the reason for this change? afaict this means the same as `^11.1.0` anyway?

Reply:
Agreed, just wanted to make the convention explicit. I will revert it right away

Reply:
updated

Reply:
thanks!